joomla/joomla-cms
# [NF] UX/UI Module menu assignment item type

UX UI

Improvment in module menu assignment to see menu item of type separator, heading, alias

### Testing Instructions
- create many menu and items (url, component, separator, alias, heading)
- create a module, go to tab Module asignment, unable to know if we check a menu items of type separator or a real url or an alias ...

Patch

- edit a module and go to tab Module asignment, now you can know the type of menu item if it is separator, alias or heading
